appveyor.yml

```diff
@@ -0,0 +1,57 @@
+version: 1.1.x.{build}
```

Comment:
This file doesn't need to be committed. This module is linux only, `appveyor.yml` is only applicable for Windows or cross-platform modules.